crates/shim-sgx/src/handler/mod.rs

```diff
@@ -75,8 +75,21 @@ pub struct Handler<'a> {
     block: &'a mut [usize],
     ssa: &'a mut StateSaveArea,
 }
-
 impl<'a> Handler<'a> {
+    fn from_access(&self, access: Access) -> Flags {
```

Comment:
Are you sure this should be a method of `Handler`? Looks like `impl From<Access> for Flags` to me.

Reply:
I'll close this as it does not exist anymore.